hq/app/logging/Cloudwatch.scala

```diff
@@ -76,8 +76,8 @@ object Cloudwatch extends Logging {
     putMetric(defaultNamespace, "Vulnerabilities", Seq(("GcpProject", project),("DataType", dataType.toString)), value)
   }
 
-  def putIamRemovePasswordMetric(reaperExecutionStatus: ReaperExecutionStatus.Value): Unit = {
-    putMetric(defaultNamespace, "IamRemovePassword", Seq(("ReaperExecutionStatus", reaperExecutionStatus.toString)), 1)
+  def putIamRemovePasswordMetric(reaperExecutionStatus: ReaperExecutionStatus.Value, value: Int = 1): Unit = {
```

Comment:
Minor: as far as I can tell this is only used in a couple of places (and value is different both times), so I probably wouldn't use a default parameter value here.